rubocop.yml

```diff
@@ -0,0 +1,1385 @@
+
```

Comment:
**[REQUIRED]**
Let's include the right linter configurations in `.rubocop.yml` file.
These are the right configs.
```
AllCops:
  Exclude:
    - "bin/*"
    - "db/**/*"
    - "config/**/*"
    - "Guardfile"
    - "Rakefile"
  DisplayCopNames: true

Metrics/LineLength:
  Max: 120
Metrics/MethodLength:
  Include: ["app/controllers/*"]
  Max: 20
Metrics/AbcSize:
  Include: ["app/controllers/*"]
  Max: 30
Metrics/ClassLength:
  Max: 150
Metrics/BlockLength:
  ExcludedMethods: ['describe']

Style/Documentation:
  Enabled: false
Style/ClassAndModuleChildren:
  Enabled: false

Layout/AlignHash:
  EnforcedColonStyle: key
Layout/ExtraSpacing:
  AllowForAlignment: false
Layout/MultilineMethodCallIndentation:
  Enabled: true
  EnforcedStyle: indented
```
Checkout this [repo](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) for the right configurations for this file.

Reply:
We used directly the ones in this repo
https://github.com/microverseinc/linters-config/tree/master/ruby